include/internal/sockets.h

```diff
@@ -97,7 +97,12 @@ typedef size_t socklen_t;        /* Currently appears to be missing on VMS */
 #   include <in.h>
 #   include <inet.h>
 #  else
-#   include <poll.h>
+#   ifdef OPENSSL_SYS_UNIX
+#     ifndef OPENSSL_SYS_TANDEM
+#      include <poll.h>
+#     endif
+#     include <errno.h>
+#   endif
```

Comment:
This just duplicates the lines 120-125 - why don't you just remove the include?

Reply:
ping @rsbeckerca

Reply:
@t8m I might be missing something. Is it just not required?

Reply:
Look at the lines: https://github.com/openssl/openssl/pull/25916/files#diff-898a9a7f3b07ce1c1d784e5452c4d55936d2e5483896953e065815579b3c6bdbL120-L125

These lines you're adding are just duplicates. Why don't you just remove `include <poll.h>`???